### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by KrisPetrienko**
- [DEMO LINK](https://KrisPetrienko.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://KrisPetrienko.github.io/layout_product-cards/report/html_report/)

+ there 2 BEM blocks and each in it own filecardstars
+ SCSS Nesting is used for , and elementsmodifierspseudo-classes
+ SCSS Variables are used for main values and placed in a separate file
+ all modifiers work as expected (Highlight first stars)stars--NN
+ Code follows all the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -6,8 +6,36 @@
...
+        <div class="product-card-menu">
+          <div class="menu-text">Price:</div>
+          <div class="menu-count">$2,199</div>
+        </div>
+        <a
+          class="product-card-button"
+           href="#"
+           data-qa="hover"
+        >BUY</a>
+      </div>
```
Comment (Andriy-Shymkiv) [inline src/index.html:38]: let's make some formatting:
```suggestion
      <div class="product-card">
        <div class="product-card-image"></div>

        <div class="product-card-text">
          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
        </div>

        <div class="product-card-code">Product code: 195434</div>

        <div class="product-card-rating">
          <div class="stars">
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
          </div>

          <div class="stars-text">Reviews: 5</div>
        </div>

        <div class="product-card-menu">
          <div class="menu-text">Price:</div>
          <div class="menu-count">$2,199</div>
        </div>

        <a
          class="product-card-button"
           href="#"
           data-qa="hover"
        >
          BUY
        </a>
      </div>
```

File: src/styles/blocks/product-card.scss [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+
+  &-image {
+    background-image: url(../images/imac.jpeg);
+    background-repeat: no-repeat;
+    width: 160px;
+    height: 134px;
+    margin-left: 3px;
```
Comment (Andriy-Shymkiv) [inline src/styles/blocks/product-card.scss:23]: not a good approach to position img
please find another way for doing this

File: src/styles/blocks/product-card.scss [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+    margin-bottom: 40px;
+    background-size: 100% 100%;
+  }
+
+  &-text {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    width: 166px;
+    height: 36px;
```
Comment (Andriy-Shymkiv) [inline src/styles/blocks/product-card.scss:33]: no need to hardcode text sizes
```suggestion
```

File: src/styles/blocks/product-card.scss [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+    .menu-count {
+      font-weight: 700;
+      font-size: 16px;
+      line-height: 18px;
+      color: $main-text-color;
+    }
+  }
+
+  &-button {
+    width: 166px;
```
Comment (Andriy-Shymkiv) [inline src/styles/blocks/product-card.scss:73]: let's use max-width instead:
```suggestion
    max-width: 166px;
    width: 100%;
```

File: src/styles/blocks/stars.scss [hunk 5]
```diff
@@ -0,0 +1,28 @@
+.stars {
+  display: flex;
+
+  &-text {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $main-text-color;
+    width: 53px;
```
Comment (Andriy-Shymkiv) [inline src/styles/blocks/stars.scss:9]: please don't
```suggestion
```

**Review by Andriy-Shymkiv — CHANGES_REQUESTED** [review]
looking good!
left a few suggestions

File: src/styles/blocks/product-card.scss (hunk 2 above)
Comment (KrisPetrienko) [reply src/styles/blocks/product-card.scss:23]: hello, I have no idea, I was tried to fix it 3 days, that's why I did it(( help me, please, with it))
image

File: src/index.html [hunk 6]
```diff
@@ -6,8 +6,46 @@
...
     <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="page" data-qa="card">
+      <div class="product-card">
+        <div class="product-card-image"></div>
```
Comment (volodymyr-soltys97) [inline src/index.html:16]: You need to add image with the img tag, it will be better

File: src/index.html [hunk 7]
```diff
@@ -6,8 +6,46 @@
...
+
+        <div class="product-card-text">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
```
Comment (volodymyr-soltys97) [inline src/index.html:20]: Need to use the semantic tag for text, fix it everywhere

File: src/index.html [hunk 8]
```diff
@@ -6,8 +6,46 @@
...
+
+        <div class="product-card-code">Product code: 195434</div>
+
+        <div class="product-card-rating">
+          <div class="stars">
```
Comment (volodymyr-soltys97) [inline src/index.html:25]: Add the modifier classes here
```suggestion
          <div class="stars stars--4">
```

File: src/index.html [hunk 9]
```diff
@@ -6,8 +6,46 @@
...
+          <div class="menu-count">$2,199</div>
+        </div>
+
+        <a
+          class="product-card-button"
+           href="#"
+           data-qa="hover"
+        >
+          BUY
+        </a>
```
Comment (volodymyr-soltys97) [inline src/index.html:47]:
```suggestion
        <a
          class="product-card-button"
          href="#"
          data-qa="hover"
        >
          BUY
        </a>
```

File: src/styles/blocks/product-card.scss [hunk 10]
```diff
@@ -0,0 +1,93 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
```
Comment (volodymyr-soltys97) [inline src/styles/blocks/product-card.scss:3]: Move font family to the html selector

File: src/styles/blocks/product-card.scss [hunk 11]
```diff
@@ -0,0 +1,93 @@
...
+}
+
+.page {
+  width: 198px;
```
Comment (volodymyr-soltys97) [inline src/styles/blocks/product-card.scss:7]:
```suggestion
  width: 200px;
```

File: src/styles/blocks/product-card.scss [hunk 12]
```diff
@@ -0,0 +1,93 @@
...
+  height: 406px;
```
Comment (volodymyr-soltys97) [inline src/styles/blocks/product-card.scss:8]: No need to hardcode height for the card
```suggestion
```

File: src/styles/blocks/product-card.scss [hunk 13]
```diff
@@ -0,0 +1,93 @@
...
+  &-text {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-text-color;
+    margin-bottom: 4px;
+  }
+
+  &-code {
+    font-weight: 400;
```
Comment (volodymyr-soltys97) [inline src/styles/blocks/product-card.scss:37]: Default value, so you can just remove it everywhere
```suggestion
```

File: src/styles/blocks/product-card.scss [hunk 14]
```diff
@@ -0,0 +1,93 @@
...
+    .menu-count {
+      font-weight: 700;
+      font-size: 16px;
+      line-height: 18px;
+      color: $main-text-color;
+    }
+  }
+
+  &-button {
+    max-width: 166px;
```
Comment (volodymyr-soltys97) [inline src/styles/blocks/product-card.scss:71]:
```suggestion
```

File: src/styles/blocks/product-card.scss [hunk 15]
```diff
@@ -0,0 +1,93 @@
...
+    width: 100%;
+    height: 40px;
+    background-color: #00ACDC;
```
Comment (volodymyr-soltys97) [inline src/styles/blocks/product-card.scss:74]: Create a variable for this color and use it everywhere

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
Let's improve your code
This part of the task is incomplete
image
image

**Pull request closed without merging**